overrides/config-overrides/expert/betterquesting/DefaultQuests.json

```diff
@@ -2297,7 +2297,7 @@
       "properties:10": {
         "betterquesting:10": {
           "autoclaim:1": 0,
-          "desc:8": "This alloy is made from §2mixing §22 Gold Dust, 1 Redstone, and 1 Glowstone Dust §2and putting it §rin your new §3Electric Blast Furnace§r.\n\nThrow §2the prepared dust into the input bus§r and in twenty seconds your §6Energetic Alloy Ingot§r will be in the output bus.\n\n§2In EV Age, you can build an Alloy Blast Smelter which doubles the yield.",
+          "desc:8": "This alloy is made from §2mixing §22 Gold Dust, 1 Redstone, and 1 Glowstone Dust §2and putting it §rin your new §3Electric Blast Furnace (EBF)§r.\n\nThrow §2the prepared dust into the input bus§r and in twenty seconds your §6Energetic Alloy Ingot§r will be in the output bus.\n\nYou might want to consider getting some §9Nitrogen§r, which speeds up EBF recipes. See the \"Basic Air Collection\" quest to start your journey. For this recipe, it reduces the duration to 13.7 seconds.\n\n§2In EV Age, you can build an Alloy Blast Smelter which doubles the yield.",
```

Comment:
Comment unrelated to changes in this PR: Are we referring to voltages as ages? I don't remember seeing this a lot.

Reply:
No we are not, let me fix that, as well as try to guide the player towards t3mm more in PGS.